src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -0,0 +1,38 @@
+package lotto.domain;
+
+import lotto.constant.ErrorMessage;
+import lotto.constant.LottoConstants;
+
+public class LottoNumber {
+    private final int number;
+
+    public LottoNumber(final int number) {
+        validateRange(number);
+        this.number = number;
+    }
+
+    public LottoNumber(final String number) {
+        int parsedNumber = validateNumber(number);
+        validateRange(parsedNumber);
+        this.number = parsedNumber;
+    }
```

Comment:
여러 생성자를 지원하는 경우 부 생성자는 주 생성자를 사용하면 좋습니다. :) 

```suggestion
    public LottoNumber(final int number) {
        validateRange(number);
        this.number = number;
    }

    public LottoNumber(final String number) {
        this(Integer.parseInt(number));
    }
```